activities/codebase-stewardship/index.md

```diff
@@ -35,7 +104,6 @@ We can provide codebase stewardship from the beginning of the development or for
 
 * [codebase in Odoo](odoo-codebase-template.md)
 * [stewardship proposal we can send to principal maintainers](stewardship-proposal-template.md)
-* [Standard for Public Code auditing template](../codebase-auditing/review-template.md)
```

Comment:
I see this is removed; is it linked from elsewhere?

Reply:
Auditing is linked several places on the page, but might indeed be good to have the template here again. I've readded it, and we can deduplicate links later.